src/main/java/games/strategy/triplea/ai/proAI/data/ProTerritory.java

```diff
@@ -291,12 +291,10 @@ public ProBattleResult getBattleResult() {
   }
 
   public String getResultString() {
-    if (battleResult == null) {
-      return "territory=" + territory.getName();
-    } else {
-      return "territory=" + territory.getName() + ", win%=" + battleResult.getWinPercentage() + ", TUVSwing="
-          + battleResult.getTuvSwing() + ", hasRemainingLandUnit=" + battleResult.isHasLandUnitRemaining();
-    }
+    return (battleResult == null)
+        ? "territory=" + territory.getName()
```

Comment:
`"territory=" + territory.getName()`
Could be extracted from the ternary operator, because it's being used in both cases.

Reply:
80f967f4d

Reply:
I actually meant something like
```java
return "territory=" + territory.getName()
  + (battleResult == null)
  ? ""
  : "territory=" + territory.getName()
  + ", win%=" + battleResult.getWinPercentage()
  + ", TUVSwing=" + battleResult.getTuvSwing()
  + ", hasRemainingLandUnit=" + battleResult.isHasLandUnitRemaining();
```
But your solution works as well, so nevermind ^^